pages/common/fnm.md

```diff
@@ -1,6 +1,6 @@
 # fnm
 
-> Fast Node.js version manager.
+> A Node.js version manager.
```

Comment:
```suggestion
> Fast Node.js version manager.
```
This change should not be included. `nvm` is simply a node version manager, while `fvm` is exactly what it says.

Reply:
but.....what? 😄  
What is different from this case to any other cases on this PR?

Reply:
If you changed this, then `nvm` and `fvm` would have essentially the same description, although nvm's [description](https://github.com/tldr-pages/tldr/blob/master/pages/common/nvm.md) is much longer.

Reply:
I think we should keep this one as-is, as the word `fast` is in `fvm`'s name.

Reply:
alright, makes sense 😄